pl_examples/domain_templates/computer_vision_fine_tuning.py

```diff
@@ -364,75 +347,60 @@ def __dataloader(self, train):
         """Train/validation loaders."""
 
         _dataset = self.train_dataset if train else self.valid_dataset
-        loader = DataLoader(dataset=_dataset,
-                            batch_size=self.batch_size,
-                            num_workers=self.num_workers,
-                            shuffle=True if train else False)
+        loader = DataLoader(
+            dataset=_dataset, batch_size=self.batch_size, num_workers=self.num_workers, shuffle=True if train else False
```

Comment:
maybe just rename the argument above to shuffle.

Reply:
I don't believe this would lead to more clarity, since setting `train` to `True` also changes the `dataset` to be used in the line above (and not only changes the shuffling).

Nontheless, `shuffle=True if train else False` could be replaced to a more simple `shuffle=train` and perhaps adding the type hints to make clear that `train` is expected to be a bool

Reply:
ok cool. missed that. yeah, `shuffle=train` would be better.

Reply:
Done in 73b34dd509c8f323ddf88b73457a22480ed99f55